src/server/src/main/java/io/cassandrareaper/core/Cluster.java

```diff
@@ -179,6 +194,15 @@ public Builder withJmxPort(int jmxPort) {
       return this;
     }
 
+    public Builder withJmxCredentials(JmxCredentials jmxCredentials) {
+      Preconditions.checkNotNull(jmxCredentials);
+      Preconditions.checkNotNull(jmxCredentials.getUsername());
+      Preconditions.checkNotNull(jmxCredentials.getUsername());
```

Comment:
was this line meant to be 
```suggestion
      Preconditions.checkNotNull(jmxCredentials.getPassword());
```